src/applications/ivc-champva/10-10D/containers/ConfirmationPage.jsx

```diff
@@ -111,13 +96,38 @@ export function ConfirmationPage(props) {
           You can print this confirmation for page for your records.
         </span>
         <br />
+        <br />
         <va-button
           uswds
           className="usa-button screen-only"
           onClick={window.print}
           text="Print this page"
         />
       </div>
+
+      <h2>What to expect next</h2>
+      <p>
+        It will take approximately 60 days to process your application once
+        received by CHAMPVA.
+        <br />
+        <br />
+        If we have any questions, need additional information, or encounter any
+        issues, we will contact you.
+      </p>
+      <h2>How to contact us about your CHAMPVA application</h2>
+      <p>
+        If you have any questions about your application you can call the
+        CHAMPVA call center at 800-733-8387. We’re here Monday through Friday,
```

Comment:
I think it's relatively common for phone numbers to be clickable. Should this be a [va-telephone](https://design.va.gov/storybook/?path=/docs/components-va-telephone--docs) component?

Reply:
Updated in [this commit](https://github.com/department-of-veterans-affairs/vets-website/commit/884ec7575eae4b79a34fb9e6c0eed8fe10a3f2d8) - thanks!